canairio/settings/testing.py

```diff
@@ -0,0 +1,19 @@
+from .base import *
+
+ENVIRONMENT = 'test'
+INFLUXDB_HOST = 'influxdb'
+try:
+    DATABASE_HOSTNAME = getenvvar('DATABASE_HOSTNAME')
+except:
+    DATABASE_HOSTNAME = 'timescaledb'
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'canairio_test',
+        'USER': getenvvar('DATABASE_USERNAME'),
+        'PASSWORD': getenvvar('DATABASE_PASSWORD'),
+        'HOST': DATABASE_HOSTNAME,
+        'PORT': '5432',
+    },
+}
```

Comment:
This would force us to maintain one file per remote environment, which is a good way to introduce infrastructure configuration issues.

Reply:
We have managed many instances in production with the approach, we can also have the port as an envvar.  If we change from Heroku to another setup it's likely necessary to change things.  You are right the port could be also an envvar

Reply:
That's not the objection I'm referring to. I'm referring to the creation of a `testing.py` file within a settings module. This introduces large amounts of unnecessary complexity.

Reply:
This one can be used with the circle-ci integratoin maybe.  For now we do not depend on timescale extension